src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/locations/LocationsMigrationService.kt

```diff
@@ -108,16 +109,18 @@ class LocationsMigrationService(
 
             nomisLocationResponse.amendments?.forEach {
               try {
-                locationsService.migrateLocationHistory(
-                  migratedLocation.id,
-                  MigrateHistoryRequest(
-                    attribute = toHistoryAttribute(it.columnName),
-                    amendedDate = it.amendDateTime,
-                    oldValue = it.oldValue,
-                    newValue = it.newValue,
-                    amendedBy = it.amendedBy,
-                  ),
-                )
+                doApiCallWithRetries {
+                  locationsService.migrateLocationHistory(
+                    migratedLocation.id,
+                    MigrateHistoryRequest(
+                      attribute = toHistoryAttribute(it.columnName),
+                      amendedDate = it.amendDateTime,
+                      oldValue = it.oldValue,
+                      newValue = it.newValue,
+                      amendedBy = it.amendedBy,
+                    ),
+                  )
+                }
```

Comment:
Also add doApiCallWithRetries to avoid network probs - locationsService.migrateLocationHistory() now ignores duplicates